collect/options.go

```diff
@@ -0,0 +1,31 @@
+package collect
+
+import (
+	"time"
+
+	"github.com/numaproj-labs/numaflow-perfman/util"
+)
+
+type Options struct {
+	step time.Duration
```

Comment:
Please add docs here for description

Reply:
added documentation to the RateInterval and Step consts in const.go